client/components/docs-example/index.jsx

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Yeah I wasn't sure where to put it.
Would you put it in a subfolder or just there in the mix with other files?